lms/djangoapps/courseware/views/views.py

```diff
@@ -152,6 +152,8 @@ def courses(request):
 
     if should_hide_master_course:
         courses = CourseOverview.objects.filter(id__in=ccx_keys)
+    elif request.user.is_staff:
```

Comment:
We don't want to break filter filter functionality for staff user. Why not just open a new if below and append invitation_only if the user is not staff? It is just a queryset until evaluated.